### ISSUE TITLE
docs: 404 not found is broken (again)

### ISSUE THREAD

**Issue by tyler36** [drud/ddev#3977]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Run a Diagnostic and Paste Link Here

n/a

### Current Behavior

I wanted to try the new symlink feature (#3904), so I went to the "latest" docs page and searched for "symlinks".
This should me a link to the "[ddev version history](https://ddev.readthedocs.io/en/latest/users/topics/version-history/)" page but when I clicked on it, I got the screen shot below.

![ddev readthedocs io_en_latest_users_topics_version-history_](https://user-images.githubusercontent.com/7234392/177914909-d01d0842-43bb-4370-b19a-032e56748869.png)

### Expected Behavior

At first, I though the version history page was bad, but after scrolling down, I realised it was a 404 page.

- [ ] fix page display on 404
- [ ] add version history page to new docs (or correct search result?)

### Steps To Reproduce

1. Visit https://ddev.readthedocs.io/en/latest/users/basics/developer-tools/
2. Search for "symlink",  
3. Click the second result, "[DDEV version history](https://ddev.readthedocs.io/en/latest/users/topics/version-history/)". It appear as the second result for me (you need to scroll down due to #3967)
4. Confirm link/page is broken

### Anything else?

Test on:

- Chrome 103
- Firefox 102

**Comment by rfay** [comment]
I removed the version history from the docs because it so often resulted in irrelevant search results. I think it might be a great clue about https://github.com/drud/ddev/issues/3967 that it even showed up in the search results, because if you look in the docs directory you won't find version_history in there any more.

**Comment by rfay** [comment]
BTW, 
* I haven't been adequately maintaining version-history.md since you fixed it up, so it wouldn't have the symlink thing in it.
* The doc was updated to include mention of that in the homeadditions page
* That feature hasn't been released yet so also wouldn't be there even if I'd been adequately maintaining version history.
* The page you want to look at is the homeadditions page, https://ddev.readthedocs.io/en/latest/users/extend/in-container-configuration/?h=homeaddition - I did find it searching for homeaddditions.

And... the 404 problem, like https://ddev.readthedocs.io/en/latest/users/extend/junk - is in fact a problem, and is another thing that I don't think was a problem running locally.
